examples/gno.land/r/users/users.gno

```diff
@@ -205,6 +205,20 @@ func GetUserByAddress(addr std.Address) *User {
 	return userI.(*User)
 }
 
+// GetAddressByInput takes a user input and try to resolve it as a user, or just return it, casted as an address.
+// This helper aims to make it easy to integrate r/users with any part of your contract that could ask a user address input.
+func GetAddressByInput(input string) std.Address {
```

Comment:
"Input" is confusing without context; how about GetUserByNameOrAddress().Address()?

Reply:
Done